Faultify.TestRunner/MutationTestProject.cs

```diff
@@ -312,13 +308,16 @@ async Task RunTestRun(IMutationTestRun testRun)
                 }
                 finally
                 {
+
                     lock (this)
                     {
                         completedRuns += 1;
+
                         sessionProgressTracker.LogTestRunUpdate(completedRuns, totalRunsCount, failedRuns);
                     }
 
-                    testProject.FreeTestProject();
+                    testProject.FreeTestProject(); //TODO: replace with deletion
```

Comment:
https://github.com/Faultify/Faultify/pull/36/files#diff-0879976499468c845e2362d93ce41c63086f5dc55d9ed7eae63ad7ae8fad7974R71

I see a todo while I also see this function. Can this function be used?